infra/bridge/src/controllers/email.js

```diff
@@ -50,44 +46,24 @@ router.post('/generate-code', emailGenerateCode, async (req, res) => {
 })
 
 router.post('/verify', emailVerifyCode, async (req, res) => {
-  if (
-    !req.session.emailAttestation ||
-    !req.session.emailAttestation.emailHash
-  ) {
-    logger.warn(`Attempted attestation for ${req.body.email} without session.`)
-    return res.status(400).send({
-      errors: ['No verification code was found for that email.']
-    })
-  }
+  const code = await getAsync(req.body.email)
 
-  const validHash = bcrypt.compareSync(
-    req.body.email,
-    req.session.emailAttestation.emailHash
-  )
-
-  if (!validHash) {
-    logger.warn(`Bad hash validation for ${req.body.email}.`)
+  if (!code) {
+    logger.warn(`Attempted attestation for ${req.body.email} without session.`)
     return res.status(400).send({
       errors: ['No verification code was found for that email.']
     })
   }
 
-  if (req.session.emailAttestation.expiry < new Date()) {
-    logger.warn(
-      `Attempted attestation for ${req.body.email} with expired code.`
-    )
-    return res.status(400).send({
-      errors: ['Verification code has expired.']
-    })
-  }
+  console.log(code)
```

Comment:
I guess this log is for debugging. Can be removed.

Reply:
Oops, thanks!